tools/build-tool/code/tooldebian.pas

```diff
@@ -0,0 +1,192 @@
+unit ToolDebian;
```

Comment:
Add our CGE copyright header, like

```
{
  Copyright 2021-2021 Eugene Loza.

  This file is part of "Castle Game Engine".

  "Castle Game Engine" is free software; see the file COPYING.txt,
  included in this distribution, for details about the copyright.

  "Castle Game Engine" is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

  ----------------------------------------------------------------------------
}
```

Reply:
Fixed in 0b9dcf3c2e0b3e4833211baa678487cc6bce9e5e